### DISCUSSION TITLE
Is it possible to forward metrics to existing grafana?

### DISCUSSION THREAD

**Discussion by matias-hick-simplisafe** [l7mp/stunner#170] [Q&A]
Hey,
I saw in the documentation that if you enable metrics, they are automatically exposed to http://:8080/metrics on each pod.
Is there a way to forward metrics to an existing Grafana? So that I can put Stunner metrics together with our other metrics.
I'd appreciate your help with this.
Best.

**Comment by levaitamas** [comment]
Hi @matias-hick-simplisafe !

The recommended way is to collect metrics with Prometheus, and visualize the Prometheus metrics in Grafana. I guess the second part (prometheus -- grafana) is done in your setup. In this case all you have to to do is to create PodMonitor to query STUNnner pods ([example](https://github.com/l7mp/stunner-helm/blob/main/helm/stunner-prometheus/templates/pod-monitor.yaml)). 

Hope this answers your question. If not, let me know. We are happy to receive :star:  and [donations](https://github.com/sponsors/l7mp) on GitHub. We are also in the consultancy business in case you need advanced [support](https://l7mp.io/#/services).

**Reply by matias-hick-simplisafe** [reply]
How do you create the podMonitor? By doing kubectl apply to that yaml?
